packages/react-reconciler/src/ReactFiber.js

```diff
@@ -388,7 +388,7 @@ export function createFiberFromElement(
   }
 
   let fiber;
-  const type = element.type;
+  let type = element.type;
```

Comment:
Why not `const`? You're not reassigning.

Reply:
Because it has fewer characters, obviously.